pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
```

Comment:
Should the function signature be changed to return an error indication? This way, you could unit test for failure cases (I don't know how you can UT panics).

Reply:
Looks like it is not too much work to propagate an error instead of panic here. Would be great to fix.

Reply:
@bowei I removed the panic. @thockin asked to return 0 instead of an error/panic.